frontend/src/pages/SaveView/SaveView.styles.scss

```diff
@@ -0,0 +1,266 @@
+.save-view-container {
+    margin-top: 70px;
+    display: flex;
+    justify-content: center;
+    width: 100%;
+
+    .save-view-content {
+        width: calc(53% - 10px);  
+
+        
+        .title {
+            color: var(--bg-vanilla-100);
+            font-family: Inter;
```

Comment:
Font family is set globally, we need not have it here.

Reply:
Removed it.